README.md

```diff
@@ -202,22 +214,25 @@ turbo = auto
 ## How to run auto-cpufreq
 auto-cpufreq should be run with with one of the following options:
 
-* [monitor](#monitor)
+* [monitor](https://github.com/AdnanHodzic/auto-cpufreq/#monitor)
```

Comment:
Let's be consistent and update links to new self referencing README structure on lines 217, 220, 223, 232, 235, 247 & 253